shukla/README.md

```diff
@@ -0,0 +1,22 @@
+1. Forked repository, cloned onto local device
+
+2. Download VSCode for Ubuntu Virtual Machine.
```

Comment:
Along the lines of https://github.com/AFIDSI/student-developer-setup/pull/8#pullrequestreview-1321241145 it would be good to get a bit more information on what the IDE setup is like. Linking to existing docs from VS Code is good too.